### ISSUE TITLE
Quad LFO CV2 functionality

### ISSUE THREAD

**Issue by bitfetcher** [bitfetcher/bF#2]
It's not clear from the few demos (of the A-132-9) what the CV2 knob does if no input is connected to CV2. I presume it's a +-5v attenuverter (scale) applied to the CV2 input that is added to the frequency, but when no cable is connected to CV2, is it disabled, or just a -5 to +5v value added to the output.

**Comment by phdsg** [comment]
this one? http://www.doepfer.de/a1439.htm i'll try to find some info.

**Comment by bitfetcher** [comment]
Yes, I've been working from that description (though my L/M/H scaling is still off...). Someone on the FB group has one and is going to do some comparing; and I'm sure he can help describe the behavior to me.
